### PR TITLE
Schema updates

### PR DESCRIPTION
**Pull request by jameshadfield**
Updates `augur export v2` and the relevant schemas to allow us to use features introduced in Auspice v2.26.0.

See commit messages for more details.

### CODE REVIEW

File: augur/data/schema-export-v2.json
Comment (huddlej) [inline augur/data/schema-export-v2.json:213]: This description says:

> Bounds from different legend entries must not overlap.

But in the `_add_legend` function in `export_v2.py`, a comment says:

```
# NOTE: structure already validated by schema
```

Should we note somewhere that we do not actually check for overlapping legend entries or should we add logic to the `_add_legend` function to warn the user when the bounds overlap? We seem to provide warnings for other similar validation issues like requesting a color scale for values that do not exist in the tree.

**Review by huddlej — COMMENTED** [review]
This looks good to me, @jameshadfield. I made a few clarifying comments inline, but we could probably just merge this as is, if you like.

It might be nice to add some minimal doctests to the new functions before merging even if it's just to get the practice of implementing tests in this module. We can chat about this offline though.

File: augur/export_v2.py
Comment (jameshadfield) [reply augur/export_v2.py:230]: Really interested in talking through this -- in my original implementation (which was years ago) I really liked using closures like this to both avoid having to pass small functions lots of variables, especially when recursion was involved, and to indicate clearly that this function was only used in this one place (in fact, I don't think it can be called from anywhere else, which is problematic for testing!).

Comment (jameshadfield) [reply augur/export_v2.py:230]: Resolving as this new code follows the style of the existing code, and bigger changes are out-of-scope for this PR

File: augur/export_v2.py
Comment (jameshadfield) [reply augur/export_v2.py:215]: I don't think a nested function like this [can be doctested](https://stackoverflow.com/questions/2403237/doctest-locally-defined-functions#comment2401456_2403757). I added further comments to the code added in this PR, and agree that doctests for the `set_colorings` function would be nice.

I also added some functional tests which cover the added code

File: augur/export_v2.py
Comment (jameshadfield) [reply augur/export_v2.py]: 👍 done

File: tests/functional/export_v2/auspice_config1.json
Comment (huddlej) [inline tests/functional/export_v2/auspice_config1.json:29]: This is really helpful documentation of how to use these new features! I will probably refer to these tests as I learn how to use these features.

**Review by huddlej — APPROVED** [review]
This looks excellent, @jameshadfield. The new tests helped me understand this PR, but they also provide great documentation for the future. The mocked up tree data is especially handy for similar types of tests in the future.

**Pull request merged**
